trunk/SUAVE/Input_Output/OpenVSP/get_fuel_tank_props.py

```diff
@@ -0,0 +1,178 @@
+    fuel_tanks = get_fuel_tank_tags(vehicle)
+    
+    num_slices = 100 # Slices used to estimate mass distribution from areas in OpenVSP
+    mass_props_output_file = tag + '_mass_props.txt'
+    vsp.SetComputationFileName(vsp.MASS_PROP_TXT_TYPE,mass_props_output_file)
+    print('Computing Fuel Tank Mass Properties... ')
+    vsp.ComputeMassProps(fuel_tank_set_index, num_slices)
+    print('Done')
+    
+    # Extract full tank mass properties from OpenVSP output file
+    fo = open(mass_props_output_file)
+    for line in fo:
+        prop_list = line.split()
+        try:
+            if prop_list[0] in fuel_tanks:
+                # Indices based on position in OpenVSP output (may change in the future)
+                cg_x = float(prop_list[2])
+                cg_y = float(prop_list[3])
+                cg_z = float(prop_list[4])
+                mass = float(prop_list[1])
+                vol  = float(prop_list[-1])
+                if 'center_of_gravity' not in fuel_tanks[prop_list[0]]: # assumes at most two identical tank names
+                    fuel_tanks[prop_list[0]].center_of_gravity = np.array([[cg_x,cg_y,cg_z]])
+                    fuel_tanks[prop_list[0]].full_fuel_mass    = mass
+                    fuel_tanks[prop_list[0]].volume            = vol
+                else:
+                    fuel_tanks[prop_list[0]].center_of_gravity = \
+                        (fuel_tanks[prop_list[0]].center_of_gravity+np.array([[cg_x,cg_y,cg_z]]))/2.
+                    fuel_tanks[prop_list[0]].full_fuel_mass   += mass
+                    fuel_tanks[prop_list[0]].volume           += vol                    
+                    
+        except IndexError:  # in case line is empty
+            pass
+
+    # Apply fuel tank properties to the vehicle
+    vehicle = apply_properties(vehicle, fuel_tanks)
+    
+    
+    return vehicle
+
+## @ingroup Input_Output-OpenVSP
+def apply_properties(vehicle,fuel_tanks):
+    """Apply fuel tank properties from OpenVSP to the SUAVE vehicle.
+    
+    Assumptions:
+    Fuel tanks exists in the fuselage and wings only
+
+    Source:
+    N/A
+
+    Inputs:
+    vehicle.fuselages.*.Fuel_Tanks.*.tag     [-]
+    vehicle.wings.*.Fuel_Tanks.*.tag         [-]
+    fuel_tanks.
+      tag                                    [-]
+      center_of_gravity                      [m]
+      full_fuel_mass                         [kg]
+      volume                                 [m^3]
+      
+
+    Outputs:    
+    vehicle.fuselages.*.Fuel_Tanks.*.mass_properties.
+      center_of_gravity                      [m]
+      full_fuel_mass                         [kg]
+      full_fuel_volume                       [m^3]
+    vehicle.wings.*.Fuel_Tanks.*.mass_properties.
+      center_of_gravity                      [m]
+      full_fuel_mass                         [kg]
+      full_fuel_volume                       [m^3]
+      
+
+    Properties Used:
+    N/A
+    """      
+    if 'wings' in vehicle:
```

Comment:
This if is unnecessary, all SUAVE vehicles have wings data:
https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Vehicle.py#L52